.github/workflows/ci.yml

```diff
@@ -188,7 +222,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: 22
-          cache: 'npm'
+          cache: "npm"
       - name: Install dependencies
         run: npm ci
       - name: Build storybook
```

Comment:
Since we're building storybook in this step, I believe you can use `packages/react/storybook-static/index.json` to get a list of all stories (if I remember the path right 😅) so that we don't have to depend on `build-components-json`. This also gives access to some meta information about the story (in particular tags) if that would be helpful for changing behavior of the axe test (like making sure to run access both before and after opening a menu)